test/Microsoft.DotNet.Watcher.Tools.Tests/CommandLineOptionsTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.DotNet.Watcher.Tools.FunctionalTests
+{
+    public class CommandLineOptionsTests
+    {
+        [Theory]
+        [InlineData(new object[] { new[] { "-h" } })]
+        [InlineData(new object[] { new[] { "-?" } })]
+        [InlineData(new object[] { new[] { "--help" } })]
+        [InlineData(new object[] { new[] { "--" } })]
+        [InlineData(new object[] { new string[0] })]
+        public void HelpArgs(string[] args)
+        {
+            // arrange
```

Comment:
nit: I am not sure what is the style in this repo but I personally don't find these comments useful.